mixer/adapter/stackdriver/metric/metric.go

```diff
@@ -169,7 +170,7 @@ func (b *builder) Build(ctx context.Context, env adapter.Env) (adapter.Handler,
 	}
 
 	ticker := time.NewTicker(cfg.PushInterval)
-
+	quit := make(chan int)
```

Comment:
i'm sure it doesn't matter at all, but `int` doesn't matter right?  so, this could just be `quit := make(chan struct{})` ?

Reply:
No it does not matter. Done.